scripts/clib/clib_test.py

```diff
@@ -0,0 +1,81 @@
+import os
+import argparse
+import shutil
+import subprocess
+
+def main():
+    parser = argparse.ArgumentParser(description="Test cpython library")
+    parser.add_argument("-cp", nargs=1, default="../../../cpython", required=False, help="Local cpython path.")
+    parser.add_argument("-rp", nargs=1, default="../..", required=False, help="Local RustPython path.")
+    args = vars(parser.parse_args())
+    CPYTHONPATH = args["cp"]
+    RPYTHONPATH = args["rp"]
+
+    if isinstance(CPYTHONPATH, list):
+        CPYTHONPATH = CPYTHONPATH[0]
+    if isinstance(RPYTHONPATH, list):
+        RPYTHONPATH = RPYTHONPATH[0]
+
+    liblst = list()
+
+    with open("clib_list.txt", 'r') as f:
+        for line in f.readlines():
+            line = line[:line.find('#')]
```

Comment:
```suggestion
            line = line.split('#')[0]
```

Reply:
That's a better way to implement comment. Thanks for letting me know!